website_sale_product_brand/models/product_template.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+from odoo.http import request
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.model
+    def _search_get_detail(self, website, order, options):
+        res = super()._search_get_detail(website, order, options)
+        domain = res["base_domain"]
+        # Need to get the context from request because if we get it from 'env' we will
+        # loose the brand_id key.
+        # TODO: Change this context by extending the options when odoo makes it
```

Comment:
This should change now that the PR is merged, isn't it?

Reply:
It is changed https://github.com/OCA/e-commerce/pull/648/commits/a66034ee9107a03ff9eacdab938dd5effd32bd61#diff-2bebf7bbe0ca4477095ab7d48844aa37f7e29a7e3dccf44af10be4bd5057a231L14-L22